lib/wsdl.js

```diff
@@ -1875,7 +1875,47 @@ WSDL.prototype._xmlnsMap = function() {
   return str;
 };
 
+var loading = { };
```

Comment:
Should this be a time limited thing as well, if the WSDL was imported in the last few minutes then use it, else get a new WSDL, could imagine this being a problem for someone out there, or even a way to "invalidate" the WSDL, maybe a future feature, food for thought